crates/consensus/src/transaction/eip1559.rs

```diff
@@ -1,6 +1,6 @@
 use crate::{SignableTransaction, Signed, Transaction, TxType};
 use alloy_eips::eip2930::AccessList;
-use alloy_primitives::{keccak256, Bytes, ChainId, Signature, TxKind, U256};
+use alloy_primitives::{aliases::TxNonce, keccak256, Bytes, ChainId, Signature, TxKind, U256};
```

Comment:
`TxNonce` is now exported correctly see but unfortunately not part of the `0.7.6` release: https://github.com/alloy-rs/core/pull/655

Reply:
Not relevant anymore